check-application-health-after-outage.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Check application health after an outage
+
+description: Describes aspects of your application you can check to ensure they're up and running after an outage.
+
+keywords: openshift, devops, outage, recovery
+
+page purpose: Gives guidelines on what to check on in an application if there has a platform-wide outage.
+
+audience: developer
+
+author: Jonathan Bond
+
+content owner: Cailey Jones
+---
+
+# Post-Outage Application Checklist
+
+One of the benefits of the OpenShift platform is a greatly reduced chance of significant unexpected outages and the near-elimination of scheduled outages. The platform is adaptable and recoverable in a way that legacy architecture is not and this lets the platform recover from hardware issues invisibly in many cases.
+
+Still, it's always possible that an outage affects the platform and brings down the applications hosted there.
+
+If this happens, speedy recovery is important. Use these guidelines to ensure your application recovers quickly and effectively.
```

Comment:
```suggestion
If this happens, speedy recovery is important. Use these guidelines to ensure your application recovers quickly and effectively.

**Note: These guidelines assume that you have built your application in a cloud-native, highly-resilient manner that makes effective use of the strengths of the OpenShift platform.  Please make sure you follow our Resiliency Guidelines (insert link here). If your application is not cloud-native in its design, it will not be capable of benefiting from the higher adaptability and recoverability of the platform mentioned above, and may require additional work to recover from a large outage, beyond what is mentioned here.**
```